tests/integ/test_byo_estimator.py

```diff
@@ -91,6 +91,7 @@ def test_byo_estimator(sagemaker_session, region, cpu_instance_type, training_se
         predictor = model.deploy(1, cpu_instance_type, endpoint_name=job_name)
         predictor.serializer = _FactorizationMachineSerializer()
         predictor.deserializer = sagemaker.deserializers.JSONDeserializer()
+        predictor.content_type = "application/json"
```

Comment:
In my next PR, I'm going to add serializer and deserializer parameters to `deploy`.